packages/gamut-labs/src/EmptySection/index.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Box,
+  FillButton,
+  FillButtonProps,
+  FlexBox,
+  Pattern,
+  Text,
+} from '@codecademy/gamut';
+import { IllustrationProps } from '@codecademy/gamut-illustrations';
+import { colors, pxRem, themed } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+export type EmptySectionProps = {
+  bodyText: string;
+  button: FillButtonProps;
+  headingText: string;
+  illustration: React.ComponentType<IllustrationProps>;
+  illustrationPosition: 'left' | 'right';
+  innerBGColor: keyof typeof colors;
+};
+
+const Dots = styled(Pattern)`
+  position: absolute;
+  top: 0;
+  right: -1rem;
+  height: calc(100% + 8rem);
+  width: calc(100% + 2rem);
+  ${themed('breakpoints.xs')} {
+    width: calc(100% + 4rem);
+    right: -2rem;
+  }
+  ${themed('breakpoints.sm')} {
+    width: calc(100% + 8rem);
+    right: -4rem;
+  }
+  ${themed('breakpoints.md')} {
+    width: calc(100% + 4rem);
+    right: 0;
+  }
+  ${themed('breakpoints.lg')} {
+    width: calc(100% + 6rem);
+  }
+`;
+
+const Title = styled(Text)`
+  margin-bottom: 0.5rem;
+`;
+
+type IllustrationContainerProps = {
+  illustrationPosition: string;
+};
+
+const IllustrationContainer = styled.div<IllustrationContainerProps>`
+  margin: 0 auto 2rem;
+  ${themed('breakpoints.md')} {
+    margin: ${({ illustrationPosition }) =>
+      illustrationPosition === 'right' ? '0 0 0 6rem' : ' 0 6rem 0 0'};
+  }
+`;
+
+export const EmptySection: React.FC<EmptySectionProps> = ({
+  bodyText,
+  button,
+  headingText,
+  illustration: Illustration,
+  illustrationPosition = 'right',
+  innerBGColor,
+}) => {
+  const direction = illustrationPosition === 'right' ? 'row-reverse' : 'row';
+
+  const renderButton = () => {
+    const { text, ...buttonProps } = button;
+    return <FillButton {...buttonProps}>{text}</FillButton>;
+  };
+
+  return (
+    <Box position="relative">
```

Comment:
It might make sense to try the background component here so everything will still work if the background color ends up being dark :)

```suggestion
import { Background } from '@codecademy/gamut-styles';
        
<Background bg={innerBGColor} position="relative">
```

https://gamut.codecademy.com/storybook/?path=/docs/foundations-colormode--page#background-